shared/subprocess/manager.go

```diff
@@ -4,19 +4,45 @@
 package subprocess
 
 import (
+	"io"
 	"io/ioutil"
+	"os"
 
 	"github.com/pkg/errors"
 	"gopkg.in/yaml.v2"
 )
 
 // NewProcess is a constructor for a process object. Represents a process with argument config. Returns an address to process
 func NewProcess(name string, args []string, stdoutPath string, stderrPath string) (*Process, error) {
+	var stdout, stderr io.WriteCloser
+	var err error
+	// Setup output capture.
+	if stdoutPath != "" {
+		stdout, err = os.Create(stdoutPath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "Unable to open stdout file")
```

Comment:
We're moving away from `errors.Wrapf` please can you use `fmt.Errorf("Unable to open stdout file: %w", err).

Also we should probably indicate which file couldn't be opened, so return:

```
fmt.Errorf("Unable to open stdout file %q: %w", stdoutPath, err)
```

Reply:
Fixed.